drivers/hmis/app/models/hmis/role.rb

```diff
@@ -384,6 +384,22 @@ def self.permissions_with_descriptions
           'Administrative',
         ],
       },
+      can_view_client_alerts: {
+        description: 'Grants the ability to view Client Alerts',
+        administrative: true,
+        access: [:editable],
```

Comment:
I don't know if this category is actually used anywhere anymore, but lets keep it consistent anyway. This perm doesn't grant any edit access so its considered viewable
```suggestion
        access: [:viewable],
```